contracts/eosio.msig/src/eosio.msig.cpp

```diff
@@ -16,12 +23,17 @@ void multisig::propose( ignore<name> proposer,
    std::vector<permission_level> _requested;
 
    _ds >> _proposer >> _proposal_name >> _requested;
```

Comment:
We can just get rid of the `ignore` for the first three arguments and simplify this part of the code.

Reply:
https://github.com/EOSIO/eosio.contracts/blob/ee3cdb17f330aed66e54ef51aa839e4af7c104c6/contracts/eosio.msig/src/eosio.msig.cpp#L42-L90